### DISCUSSION TITLE
cache album art, infos etc over server not local

### DISCUSSION THREAD

**Discussion by GekoCH** [music-assistant/support#1838] [Q&A] [answered]
hy

Is it correct that, when accessing my music-assistant server from a tablet, it will download album art and other related content from the server if available? If not, does it attempt to retrieve the art and other data over the tablet's internet connectivity?

I'm inquiring because I own a Xiaomi tablet that I use to manage my home. However, this tablet is restricted from accessing the internet by my firewall, allowing only local access. When I access my music-assistant server through the local IP, I can reach it, and all images are displayed if they have been cached beforehand. Additionally, browsing Tidal works, but without any accompanying images.

Is it possible to route all traffic over the main music-assistant server an not the client itself?

Andy

**Comment by OzGav** [comment]
@marcelveldt ??

**Reply by marcelveldt** [reply, marked as answer]
We are currently using a free online image resize service for the image thumbs to cause less stress to the server. I guess we can also use the server's builtin image resizer at all times, we'll just have to test the impact of that. Can you convert this one to issue on the backlog please @OzGav ?

**Reply by OzGav** [reply]
Progress has been made for this in b137 with some further improvements to come
